pocs/focuser/focuser.py

```diff
@@ -398,6 +360,22 @@ def _autofocus(self,
                                               self._camera.uid,
                                               start_time)
 
+        if take_dark:
+            file_path = "{}/{}.{}".format(file_path_root, "dark", self._camera.file_extension)
```

Comment:
Does this need a unique (timestamped?) name?

Reply:
Maybe. It does have a unique path, though, which includes both the camera UID and a timestamp. Have been doing it this way since the autofocus functionality was added.